config/config.exs

```diff
@@ -31,5 +31,5 @@ use Mix.Config
 
 config :gym_tcp_api,
   port: 4040,
-  worker: 4,
-  distributed: false
\ No newline at end of file
+  worker: 2,
+  distributed: true
```

Comment:
I think most users don't use a distributed setup (running more than one machine), so I would set it to false per default. It's slightly faster, since you don't have to spawn another process.